packages/x-charts-pro/src/Heatmap/HeatmapItem.tsx

```diff
@@ -110,7 +108,6 @@ function HeatmapItem(props: HeatmapItemProps) {
   const Cell = slots?.cell ?? HeatmapCell;
   const cellProps = useSlotProps({
     elementType: Cell,
-    additionalProps: interactionProps,
```

Comment:
Isn't this also technically a breaking change? We're removing the interaction item props from a potential slot, which means that it might break someone's behavior. 

For example, if they're using composition using a custom `HeatmapPlot` and rendering a `HeatmapItem` with slots, it might break. Seems a bit of an edge case, though